src/engine/test/assets/decoders/json.yml

```diff
@@ -1,7 +1,7 @@
 name: json
 
 check:
-  - event.original: +exists
+  - event.original: +r_match/^{.+}$
```

Comment:
Avoid using a regex, it is almost as likely to have a key at the beginning as the one at the end, you can check only the key at the beginning is `+s_start/{`